packages/rhi-webgl/src/GLTexture.ts

```diff
@@ -124,6 +124,11 @@ export class GLTexture implements IPlatformTexture {
           internalFormat: GLCompressedTextureInternalFormat.RGBA_BPTC_UNORM_EXT,
           isCompressed: true
         };
+      case TextureFormat.BC6H:
+        return {
+          internalFormat: gl.getExtension("EXT_texture_compression_bptc").COMPRESSED_RGB_BPTC_UNSIGNED_FLOAT_EXT,
```

Comment:
you can use `GLCompressedTextureInternalFormat.RGB_BPTC_UNSIGNED_FLOAT_EXT`

Reply:
Why `GLCompressedTextureInternalFormat` is deprecated?